vspace/src/main/java/edu/asu/diging/vspace/web/staff/ExhibitionConfigurationController.java

```diff
@@ -99,16 +109,123 @@ public RedirectView createOrUpdateExhibition(HttpServletRequest request,
+            @RequestParam("spaceLinkImage") MultipartFile spaceLinkImage,
+            RedirectAttributes attributes) throws IOException {
+        Exhibition exhibition;
+        if (exhibitID == null || exhibitID.isEmpty()) {
+            exhibition = (Exhibition) exhibitFactory.createExhibition();
+        } else {
+            exhibition = (Exhibition) exhibitManager.getExhibitionById(exhibitID);
+        }
+        
+        IVSImage spaceDefaultImage = spaceLinkImage != null ? 
+                imageService.storeImage(spaceLinkImage.getBytes(), spaceLinkImage.getOriginalFilename()) : null; 
+        
+        exhibition.setSpaceLinkDefaultImage(spaceDefaultImage);
+        exhibition = (Exhibition) exhibitManager.storeExhibition(exhibition);
+        attributes.addAllAttributes(addSuccessAttributes(exhibition));
+
+        return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
+    }
+    
+    /**
+     * To create or update the default module link image
+     * 
+     * @param exhibitID - the ID of the exhibition to update
+     * @param moduleLinkImage - the MultipartFile representing the module link image
+     * @param attributes - the RedirectAttributes object
+     * @return a RedirectView to the exhibition configuration page
+     */
+    @RequestMapping(value = "/staff/exhibit/config/moduleLinkImage", method = RequestMethod.POST)
+    public RedirectView createOrUpdateModuleLinkImage(HttpServletRequest request,
+            @RequestParam(required = false, name = "exhibitionParam") String exhibitID,
+            @RequestParam("moduleLinkImage") MultipartFile moduleLinkImage,
+            RedirectAttributes attributes) throws IOException {
+        Exhibition exhibition;
+        if (exhibitID == null || exhibitID.isEmpty()) {
+            exhibition = (Exhibition) exhibitFactory.createExhibition();
+        } else {
+            exhibition = (Exhibition) exhibitManager.getExhibitionById(exhibitID);
+        }
+        
+        IVSImage moduleDefaultImage = moduleLinkImage != null ? 
+                imageService.storeImage(moduleLinkImage.getBytes(), moduleLinkImage.getOriginalFilename()) : null; 
+        
+        exhibition.setModuleLinkDefaultImage(moduleDefaultImage);
+        exhibition = (Exhibition) exhibitManager.storeExhibition(exhibition);
+        attributes.addAllAttributes(addSuccessAttributes(exhibition));
+        
+        return new RedirectView(request.getContextPath() + "/staff/exhibit/config");
+    }
+    
+    /**
+     * To create or update the default external link image 
+     * 
+     * @param exhibitID - the ID of the exhibition to update
+     * @param externalLinkImage - the MultipartFile representing the external link image
+     * @param attributes - the RedirectAttributes object 
+     * @return a RedirectView to the exhibition configuration page
+     */
+    @RequestMapping(value = "/staff/exhibit/config/externalLinkImage", method = RequestMethod.POST)
```

Comment:
It doesn't seem like there are 3 different methods needed, are there? Only one line is different, isn't it?